composer.json

```diff
@@ -80,10 +80,6 @@
             "drupal/field_group": {
                 "https://www.drupal.org/project/field_group/issues/2787179#comment-13467953": "https://www.drupal.org/files/issues/2020-02-17/2787179-highlight-html5-validation-45.patch"
             },
-            "drupal/inline_entity_form": {
```

Comment:
These changes exist in 1.0-rc8 but doesn't exist in minor version that currently is 1.0-rc7.
Might be we also should update minimum version?
`"drupal/inline_entity_form": "~1.0-rc8"`

Reply:
Done, thanks!